sematic/db/migrations/20230221004041_swap_users.sql

```diff
@@ -0,0 +1,23 @@
+-- migrate:up
+
+ALTER TABLE users RENAME TO users_backup;
+
+ALTER TABLE users_tmp RENAME TO users;
+
+DROP TABLE users_backup;
+
+-- migrate:down
+
+ALTER TABLE users RENAME TO users_tmp;
+
+CREATE TABLE users (
+    email TEXT NOT NULL,
+    first_name TEXT,
+    last_name TEXT,
+    avatar_url TEXT,
+    api_key TEXT NOT NULL,
+    created_at timestamp NOT NULL,
+    updated_at timestamp NOT NULL,
+
+    PRIMARY KEY (email)
+);
```

Comment:
This might be a little debatable, but if we don't drop table on line 7, we can recover data in the rollback step instead of creating an empty table. I don't feel strongly. Just wanna mention an option.

Reply:
We shouldn't keep unexpected tables in the DB, for hygiene.
This might even have security consequences in the future.